semantic_release/vcs_helpers.py

```diff
@@ -120,8 +120,13 @@ def commit_new_version(version: str):
     """
 
     check_repo()
+    commit_subject = config.get('semantic_release', 'commit_subject')
     commit_message = config.get('semantic_release', 'commit_message')
-    message = '{0}\n\n{1}'.format(version, commit_message)
+
+    message = commit_subject.format(version=version)
+    if commit_message:
```

Comment:
There is one case which is not _exactly_ the same here: any user which previously set the `commit_message` as `""` would technically get a commit of

```
{version}

```

Here, we check to see if the commit message is empty: if it is not, only then would the two newlines be added. So this is the real difference, it's not technically backwards compatible but I think it is an improvement. If you feel you don't want to include it until 5.0, you could just remove the `if` statement.
```